.github/workflows/backend-test.yml

```
@@ -0,0 +1,21 @@
+name: backend-test
+
+on:
+  push:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  backend-test:
+    runs-on: ubuntu-latest
+    steps:
```

It would be nice to do a matrix here, since we're supporting local development on multiple OS, something like:

```suggestion
jobs:
  backend-test:
    strategy:
      matrix:
        go-version: [1.16.x]
        os: [macos-latest, ubuntu-latest, windows-latest]
    runs-on: ${{ matrix.os }}
    steps:
```
The go-version can be used below in set up golang to future proof supporting other version, just add them to the matrix.